src/client/proxy/socks.h

```diff
@@ -334,9 +340,15 @@ class SOCKSHandler
       SOCKSServer* parent,
       std::shared_ptr<boost::asio::ip::tcp::socket> socket);
 
-  ~SOCKSHandler();
+  ~SOCKSHandler()
+  {
+    Terminate();
```

Comment:
No, move back. These are not small functions. This is noted in https://google.github.io/styleguide/cppguide.html#Inline_Functions.

Reply:
Alright, I misunderstood again. Didn't realize that the line count applied through the inner function call.

Reply:
Always, because that is the code path.